gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -82,12 +82,13 @@ def add_arguments(self, parser):
               'Note that each VCF file must still contain valid header files '
               'even if this is provided.'))
     parser.add_argument(
-        '--infer_undefined_headers',
+        '--infer_headers',
```

Comment:
Please keep both versions to make this change backwards compatible. You can just add both like this:
```
parser.add_argument(
    '--infer_headers',
    '--infer_undefined_headers',
    type = 'bool', ...

```